index.html

```diff
@@ -9,19 +9,20 @@
   </head>
 
   <body>
-    <header>
+    <header class="header">
```

Comment:
if you use the same styling everywhere in the header, you don't need to define it as a class, unless you have more styling planned for later :)